index.js

```diff
@@ -2,13 +2,16 @@
 
 "use strict";
 const generator = require("./critical");
+const fs = require('fs');
 const url = process.env.URL;
 const dimensions = process.env.DIMENSIONS || "1300x900";
 const location = process.env.LOCATION || "/output/critical.css";
 
 generator
   .generateCritical(url, dimensions, location)
-  .then((css) => {})
+  .then((css) => {
```

Comment:
I explored this option but since we're handling file writing with critical, better to stick with that.